config/sample_manifest_excel/ranges.yml

```diff
@@ -74,3 +74,11 @@ insdc_country:
   identifier: :name
   scope: :sorted_for_select
   scope_on: Insdc::Country
+collected_by:
+  options:
+    - UKB_Reading_11026
+    - UKB_Newcastle_11027
+    - UKB_Stockport_11025
+    - UKB_Bristol_11028
+    - ELGH_London_21032
+    - ELGH_Bradford_21034
```

Comment:
Ah, I was hoping that it would be possible to provide this list specifically for the Cardinal manifest (because the values `UKB_Reading_11026`, etc. are only relevant for the Cardinal project, and other projects might want different values). Do you know if that is possible? I wonder whether the `scope` thing I can see on a few of the fields is relevant?